### PR TITLE
SEM-95] FORM FIELDS FORM USER MANAGEMENT 

### PR DESCRIPTION
**Pull request by JorgePachecoCordova**
Description
We need to implement a component that contains the elements shown in the image, thus containing all the form fields, as well as integrating the editing component shown in the image.

Acceptance Criteria
Integrate the component into the appropriate section shown in the images.
Ensure that icons and labels are aligned and sized correctly as shown in the screenshots.

Color Variants
See figure.

Implementation in Storybook:
Create entries in Storybook for each button to show the existing states of the component.
Include controls to allow testing of different states and properties.

Test Driven Development (TDD):
Write unit tests for both buttons to verify navigation and state changes.
Ensure that tests cover color changes on hover and alignment of icons and labels.
image

### CODE REVIEW

File: src/ui/components/molecules/use-form/UserForm.test.tsx [hunk 1]
```diff
@@ -0,0 +1,36 @@
+import { render, screen, fireEvent } from '@testing-library/react';
+import { describe, it, expect } from 'vitest';
+import UserForm from './UserForm';
+
+describe('UserForm Component', () => {
+  it('renders all form fields with correct placeholders', () => {
+    render(<UserForm onChange={() => {}} label="Test Label" />);
+
+    // Verificar que todos los campos tienen sus placeholders
```
Comment (Ditmar) [inline src/ui/components/molecules/use-form/UserForm.test.tsx:9]: remove the comments

File: src/ui/components/molecules/use-form/UserForm.tsx [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+      </div>
+
+      <div className={styles.form__field}>
+        <span className={styles.form__label}>Password</span>
+        <Placeholder placeholder="Enter your password" onChange={onChange} />
+      </div>
+
+      <div className={styles.form__field}>
+        <span className={styles.form__label}>Role</span>
+        <Placeholder placeholder="Enter your role" onChange={onChange} />
```
Comment (Ditmar) [inline src/ui/components/molecules/use-form/UserForm.tsx:25]: Great!

**Review by Ditmar — CHANGES_REQUESTED** [review]
good job please put the url of the storybook to check the component

File: src/ui/components/molecules/use-form/UserForm.tsx (hunk 2 above)
Comment (JorgePachecoCordova) [reply src/ui/components/molecules/use-form/UserForm.tsx:25]: https://ditmar.github.io/backoffice-openscience/branch-SEM-95/?path=/story/ui-components-molecules-user-form--default
story

**Review by Ditmar — APPROVED** [review]

**Pull request merged**